src/applications/facility-locator/components/ResultsList.jsx

```diff
@@ -50,24 +50,14 @@ export class ResultsList extends Component {
     }
   }
 
-  visNEightIsShowNumber(result, query, facilityLocatorShowHealthConnectNumber) {
+  isHealthAndHealthConnect(result, query) {
     let final = false;
     if (
-      result?.attributes?.visn === '8' &&
       query?.facilityType === 'health' &&
-      facilityLocatorShowHealthConnectNumber
-    ) {
-      final = facilityLocatorShowHealthConnectNumber;
-    } else if (
-      result?.attributes?.visn === '2' &&
-      query?.facilityType === 'health'
-    ) {
-      final = true;
-    } else if (
-      result?.attributes?.visn === '17' &&
-      query?.facilityType === 'health'
+      result?.attributes?.phone?.healthConnect !== null
     ) {
       final = true;
+      return final;
```

Comment:
This early return isn't doing much - the return on line 62 has to be there, so I'd suggest just leaning on that one & avoid the extra line of code here.